pay-api/tests/docker/docker-compose.yml

```diff
@@ -22,18 +22,6 @@ services:
       retries: 10
     volumes:
       - ./setup:/tmp/keycloak/test/
-  nats:
```

Comment:
Future: Maybe we need a docker container for pubsub? I think it's mostly HTTP requests (push)?